yarn.lock

```diff
@@ -2568,22 +2568,52 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@polkadot/api-contract@npm:^7.0.2":
-  version: 7.0.2
-  resolution: "@polkadot/api-contract@npm:7.0.2"
+"@polkadot/api-base@npm:7.0.3-1":
+  version: 7.0.3-1
+  resolution: "@polkadot/api-base@npm:7.0.3-1"
   dependencies:
     "@babel/runtime": ^7.16.5
-    "@polkadot/api": 7.0.2
-    "@polkadot/types": 7.0.2
-    "@polkadot/types-create": 7.0.2
+    "@polkadot/rpc-core": 7.0.3-1
+    "@polkadot/types": 7.0.3-1
+    "@polkadot/util": ^8.2.2
+    rxjs: ^7.4.0
+  checksum: 8e6d7264d89650b31e88c778b6ac3cbd979d9e0db2973d7adad01982012e6e493766575a422f6c16ca4ea07a03302b5e86ba3541c95f74ae791f039b4d903f53
+  languageName: node
+  linkType: hard
+
+"@polkadot/api-contract@npm:^7.0.3-0":
```

Comment:
Somewhere things got mismatched between the versions here - the `./dev/all-update.sh` script will ensure there is a single version and dedupe it as well. (The lockfile here is weird with the `-0` & `-1` matching)

Reply:
what i did is to bump the versions in `package.json` and then run `yarn`

Reply:
Ahh, yes, that is the issue - they are in multiple places in the root and in the workspace level (not actually enough, most pages do still pull in transient deps).